src/plugin.ts

```diff
@@ -65,6 +65,16 @@ const recommendedRules = {
 	),
 } satisfies Linter.RulesRecord;
 
+const recommendedPublishableRules = {
+	...recommendedRules,
+	...Object.fromEntries(
+		Object.entries(rules)
+			// eslint-disable-next-line @typescript-eslint/no-deprecated
+			.filter(([, rule]) => rule.meta.docs?.category === "Publishable")
```

Comment:
> deprecated

[Non-Actionable] _Yeah_, at some point we'll probably want to figure this out. In other packages I've used `meta.docs.recommended`. But per our other discussions (...somewhere, I don't have the reference handy right now) this is definitely fine for now pending more strong guidance from core.